src/command_modules/azure-cli-network/azure/cli/command_modules/network/commands.py

```diff
@@ -840,6 +841,10 @@ def _make_singular(value):
         g.custom_command('create', 'create_vnet_gateway_root_cert')
         g.custom_command('delete', 'delete_vnet_gateway_root_cert')
 
+    with self.command_group('network vnet-gateway ipsec-policy', network_vgw_sdk, min_api='2018-02-01') as g:
+        g.custom_command('add', 'add_vnet_gateway_ipsec_policy', supports_no_wait=True, doc_string_source='IpsecPolicy')
```

Comment:
Comment: Didn't know that you could reference a model to use its doc string for help. I am guessing this is a knack feature?

Reply:
I actually think this is a CLI-only thing and it is pretty wonky.  It was a workaround for a pattern that existed prior to the Knack conversion.

Reply:
Ah I see, would we want to document this or not (seeing its wonky)?

Reply:
No. I wouldn't encourage anyone to use it.  I only did because it was a copy-paste job for these commands.